### PR TITLE
6.View Article Commments - Comments added

### PR DESCRIPTION
**Pull request by bobbynagra92**

### CODE REVIEW

File: src/Components/SingleArticle.jsx [hunk 1]
```diff
@@ -1,34 +1,84 @@
...
+              return (
+                <Toast key={comment.comment_id} className='comment'>
+                  <Toast.Header closeButton={false}>
+                    <img
+                      src={avatar_url}
+                      className='user_avatar'
+                      alt={`${comment.author} user avatar`}
+                    />
+                    <h6>{comment.author}</h6>
+                    <small>{formatDate(comment.created_at)}</small>
```
Comment (callumu21) [inline src/Components/SingleArticle.jsx:71]: Nice use of a util function

File: src/Components/SingleArticle.jsx [hunk 2]
```diff
@@ -1,34 +1,84 @@
...
+      <h6>Written by {matchUserAndAuthor(article.author, users).name}</h6>
+      <h6>Created: {formatDate(article.created_at)}</h6>
+      <p id='article-body'>{article.body}</p>
+      {commentLoading ? (
+        <Loading isLoading={commentLoading} />
+      ) : (
+        <>
+          <h3>Comments 💬</h3>
+          <div className='comments'>
+            {comments.map((comment) => {
```
Comment (callumu21) [inline src/Components/SingleArticle.jsx:57]: This is great if an article has comments, but you should also consider what you'd like a user to see if there aren't any to display (i.e. a new article that hasn't been commented on yet). I'd have an alternative display here that is conditionally rendered if the comments array is empty

**Review by callumu21 — CHANGES_REQUESTED** [review]
Nice job, Bobby. Just one thing that needs adding before this can be approved:

1. If an article has no associated comments, you currently won't render anything to the user. I'd add some conditional rendering that displays a message like 'This article has no comments' if the comments array is empty.

Just an extra small note, the avatar image on medium screen sizes and above is squashed so you may want to look at resizing it to look as good as on smaller screens.

Once you've added in the display for no comments, resubmit this PR and we can take another look 👍

**Review by robert112233 — APPROVED** [review]
This change gives the user a better experience and doesn't leave them in the dark about there not being comments. Well done!

**Pull request merged**